### PR TITLE
feat:update mode UI

### PR DESCRIPTION
**Pull request by thoa01**

### CODE REVIEW

File: the-telephone-app-fe/src/assets/index.css [hunk 1]
```diff
@@ -384,4 +384,26 @@ input {
...
+  background-color: var(--secondary-color);
+  border-radius: 8px;
+  margin: 12px;
+}
+
+.img-mode {
+  width: 40%;
+  height: 40%;
+  border-radius: 12px;
+}
```
Comment (ces-manhnguyen) [inline the-telephone-app-fe/src/assets/index.css:399]: add empty line after this

File: the-telephone-app-fe/src/components/JoinRoom/JoinRoom.js [hunk 2]
```diff
@@ -186,20 +187,41 @@ const JoinRoom = () => {
           <div className="col-1 section-sub"></div>
           <div className="col-7 section">
             <div className="row h-80">
-              <div className="col-6">
-                <div>
-                  <label className="button">
-                    <input type="radio" name="mode" value="IN_PROGRESS" checked = {selectedMode === "IN_PROGRESS"} onChange={handleModeChange} />
-                    NORMAL
-                  </label>
+              <div className="row h-13 align center text-title">MODE</div>
```
Comment (ces-manhnguyen) [inline the-telephone-app-fe/src/components/JoinRoom/JoinRoom.js]: SELECT MODE is clearer

File: the-telephone-app-fe/src/components/JoinRoom/JoinRoom.js [hunk 3]
```diff
@@ -10,9 +10,10 @@ import { useLocation } from "react-router-dom";
 import { useNavigate } from "react-router-dom";
 import "react-confirm-alert/src/react-confirm-alert.css";
 import imgLogo from "../../assets/logo.png";
+import imgWriteMode from "../../assets/ww-removebg-preview.png";
+import imgPaintMode from "../../assets/Canvas-616x750-removebg-preview.png";
```
Comment (ces-manhnguyen) [inline the-telephone-app-fe/src/components/JoinRoom/JoinRoom.js]: rename png files with condensed and clear names.

**Comment by ces-manhnguyen** [comment]
Add space after feat: in commit message

**Review by ces-manhnguyen — APPROVED** [review]

**Review by DanHuy77 — APPROVED** [review]

**Pull request merged**